### ISSUE TITLE
Broken overmap

### ISSUE THREAD

**Issue by 5ENP4I** [CleverRaven/Cataclysm-DDA#74973]
### Describe the bug

Cannot use overmap, try to open it but "you have no idea where you are" message in log appears. Game probably thinks player is on the lost level (where message always appears instead of map) but player is overground in "normal" circumstances.

### Attach save file

[DreamyNightmare-trimmed.tar.gz](https://github.com/user-attachments/files/16169007/DreamyNightmare-trimmed.tar.gz)

### Steps to reproduce

1. upon loading game, if you push button to open overmap, the message from lost level appears in log.
2. walk a little (probably go to other overmap tile) and you should se "normal" Cataclysm behavior.

### Expected behavior

player should be able to use overmap there, where it is possible.

### Screenshots

screnshot of lost level messages
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/81872136/7761358a-5f8f-4a51-b0c2-bbe006bd4348)
screenshot of normal behaviour numbers of turns after
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/81872136/156a9974-4560-4bc0-a5b5-8fe1b931ef9b)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.3737 (23H2)
- Game Version: 06460ed [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Slowdown Fungal Growth [no_fungal_growth],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Satelites modz [eiseis_modpack]
]

### Additional context

Custom mods that rebalance game a little and are not in CleverRaven repo.
[EiSeIs_Modpack.zip](https://github.com/user-attachments/files/16169044/EiSeIs_Modpack.zip)

**Comment by KHeket** [comment]
Am I right, that you were in center of portal storm, then accept "halfway" dialog variant from "portal voice", and then you didnt be teleported in portal dungeon, and this bug appeared?

**Comment by 5ENP4I** [comment]
Not at all. It was second day after game begining, and my character has not met any portal storms before the bug (neither i went to lost level)

**Comment by meelock** [comment]
I appear to be getting this bug alongside a disappearing NPC whenever I go into a subway lab with an NPC in tow.
I also tend to get an error something like this when it happens:
DEBUG    : error: item wrapper cannot fit into pocket while loading: item is too long

 FUNCTION : combine
 FILE     : D:\a\Cataclysm-DDA\Cataclysm-DDA\src\item_contents.cpp
 LINE     : 737
 VERSION  : e418d44
